lib/watir/elements/radio.rb

```diff
@@ -9,8 +9,8 @@ def build
     # Selects this radio button.
     #
 
-    def set
-      click unless set?
+    def set(bool = true)
+      click if bool && !set?
```

Comment:
Are people going to get confused why `radio.set(false)` doesn't clear the radio button? Is it worth throwing an error if `!bool && set?`?

Reply:
I'm thinking of `#set` and `#get` as things that will primarily be used by Page Object implementations.
If you use `#populate_page_with` the `Hash` object should be able to dynamically set it to true/false. I think that's preferable to having to delete the key before passing it in.

Then again... I'd hope people would use RadioSet instead of Radio for these in general.

Reply:
and "clearing a radio button" isn't something a user does with a page, so I don't think a user should reasonably expect that to be the behavior here.